connect/runtime/src/test/java/org/apache/kafka/connect/util/clusters/EmbeddedConnectClusterAssertions.java

```diff
@@ -455,48 +455,42 @@ public void assertConnectorIsFailedAndTasksHaveFailed(String connectorName, int
     }
 
     /**
-     * Assert that a connector and its tasks are not running.
+     * Assert that a connector does not exist. This can be used to verify that a connector has been successfully deleted.
      *
      * @param connectorName the connector name
      * @param detailMessage the assertion message
      * @throws InterruptedException
      */
-    public void assertConnectorAndTasksAreNotRunning(String connectorName, String detailMessage)
+    public void assertConnectorDoesNotExist(String connectorName, String detailMessage)
```

Comment:
Given that we know this method is only being referenced from connector deletion cases, why don't we just rename it to `assertConnectorDeleted`. As such, the connector won't exist only when it's deleted? For ] cases when we want to verify if the connector has been created successfully, we will have asserts for checking that and don't need to rely on this method anyways. WDYT?

Reply:
I preferred `assertConnectorDoesNotExist` over `assertConnectorDeleted` since it's a more accurate description of what exactly is being verified.

> As such, the connector won't exist only when it's deleted

It could also never have been created in the first place right?